freqtrade/freqtradebot.py

```diff
@@ -452,14 +453,8 @@ def check_and_call_adjust_trade_position(self, trade: Trade):
         If the strategy triggers the adjustment, a new order gets issued.
         Once that completes, the existing trade is modified to match new data.
         """
-        if self.strategy.max_entry_position_adjustment > -1:
-            count_of_buys = trade.nr_of_successful_buys
-            if count_of_buys > self.strategy.max_entry_position_adjustment:
-                logger.debug(f"Max adjustment entries for {trade.pair} has been reached.")
-                return
-        else:
-            logger.debug("Max adjustment entries is set to unlimited.")
-        current_rate = self.exchange.get_rate(trade.pair, refresh=True, side="buy")
+        current_entry_rate, current_exit_rate = self.exchange.get_rates(trade.pair, True)
+        current_rate = current_entry_rate  # backward compatibilty
```

Comment:
You can save this assignment, and instead pass it to the callback directly (basically the same code - but in the function call).

Reply:
updated